dogs.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt-A-Dog</title>
+  </head>
+  <body>
+    <header>
+      <img class="logoimg" src="adopt-a-dog-spring2019-master/images/adoptadog-logo.svg" alt="adopt a dog logo"/>
+      <h1 class="headertext">Seattle's premiere dog adoption agency</h1>
+      <img class="cartimg" src="adopt-a-dog-spring2019-master/images/cart-icon.svg" alt="shopping cart icon" />
+      <form action="checkout.html">
+    <input type="submit" value="Checkout" />
+</form>
+    </header>
+
+    <div class="nav">
+      <ul>
+        <li><a href="index.html">Home /</a></li>
+        <li><a href="dogs.html">Dogs /</a></li>
+        <li><a href="blog.html">Blog</a></li>
+      </ul>
+    </div>
+
+<div class="here">
+  <h2>Here are all of our dogs!</h2>
+</div>
+
+<article class="dogprofiles">
```

Comment:
Like the other file, I'd go with section here, and article (or something) for the dogs, but I like the organizaton of this. It'll make layout a cinch!